src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -10,13 +10,13 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Someone being a little bitch? Smite them down...", usage = "/<command> [playername]")
+@CommandParameters(description = "Someone being a little bitch? Smite them down...", usage = "/<command> [playername] [reason]")
```

Comment:
So this is fine then because we want it to be optional (according to your other comment anyway...)?